ResidentInformationApi.Tests/V1/UseCase/ListContactUseCaseTests.cs

```diff
@@ -43,9 +43,9 @@ public void SetUp()
             _housingUrl = "http://HOUSING_API_URL";
             _mosaicUrl = "http://MOSAIC_API_URL";
 
-            Environment.SetEnvironmentVariable("ACADEMY_API_URL", _academyUrl);
-            Environment.SetEnvironmentVariable("HOUSING_API_URL", _housingUrl);
-            Environment.SetEnvironmentVariable("MOSAIC_API_URL", _mosaicUrl);
+            Environment.SetEnvironmentVariable("ACADEMY_API_LIST_URL", _academyUrl);
+            Environment.SetEnvironmentVariable("HOUSING_API_LIST_URL", _housingUrl);
+            Environment.SetEnvironmentVariable("MOSAIC_API_LIST_URL", _mosaicUrl);
```

Comment:
I think these need to stay in the original format, they should be the same for both endpoints and need to match what we have in the serverless.yml

Reply:
The list URIs can be stored as environment variables, but the viewrecords endpoint might need the URI constructed in the code?

Reply:
This should just be a base url, so it will be constructed on top of this. For example, the base API might be "https://hackeny-academy-api.com" then both the list endpoint and the view endpoint would use this to create the full endpoints e.g. "https://hackeny-academy-api.com/api/v1/clamiants" and "https://hackeny-academy-api.com/api/v1/clamiants/claim/12345"